docs/commcare-cloud/deploy.md

```diff
@@ -0,0 +1,58 @@
+# Deploying CommCare HQ code changes
+
+1. Update your commcare-cloud environment to make sure it's on the latest version of commcare-cloud
+```
+$ update-code
+```
+2. Connect to the VPN (if applicable)
+1. Use commcare-cloud to deploy the environment
+
+```
+$ cchq <env> fab deploy
+
+# or to run it from the control machine which usually makes it faster
+$ cchq <env> --control fab deploy
+```
+
+If you had to wait for a preindex, retry deploy when the preindex completes,
```

Comment:
Might be worth noting how to kick off a preindex ahead of time.

Reply:
updated